data/json/vehicles/carts.json

```diff
@@ -255,6 +255,18 @@
     ],
     "items": [ { "x": 0, "y": 0, "chance": 30, "items": [ "goggles_welding" ] } ]
   },
+  {
+    "id": "solar_unit",
+    "type": "vehicle",
+    "name": "mounted solar panel",
```

Comment:
what's this thing for? If it's a roof mounted solar panel, that might be better done as furniture that can be removed by Disassembling it from the roof?